media/css/style.css

```diff
@@ -12,7 +12,6 @@
 
 GtkScrolledWindow, GtkViewport {
     border-width: 0px;
-    -GtkWidget-focus-line-width: 0px;
```

Comment:
Does replacing that line with the option below help?
```
outline-width: 0px;
```
https://developer.gnome.org/gtk3/stable/GtkWidget.html#GtkWidget--s-focus-line-width